docker/init_scripts/init

```diff
@@ -115,19 +115,43 @@ watchdog_process_pid() {
 	fi
 }
 
+stop_process_pid() {
+	PROCESS=$1
+  if [[ -f "/tmp/${PROCESS}.pid" ]]; then
+    PID=$(cat "/tmp/${PROCESS}.pid") || true
+		if [[ -d "/proc/${PID}" ]]; then
+		  info_log "stopping ${PROCESS}"
+		  kill "${PID}" || true
+		  # wait for process exit
+		  while [ -e "/proc/${PID}" ]; do sleep 0.1; done
+    fi
+  fi
+}
+
+shutdown() {
+  # shutdown in reverse order
+  stop_process_pid sheduler
```

Comment:
typo: `sheduler` instead of `scheduler`

Reply:
nice catch, thanks, this is why last one `python3` process been killed, now termination exit codes looks like
```
python3          163192 162602 163192 15.15   0
python3          163184 162602 163184 15.39   0
python3          163176 162602 163176 15.45   signal 15 (TERM)
nginx            163193 163163 163193 15.58   0
nginx            163190 163163 163190 15.58   0
nginx            163173 163163 163173 15.58   0
nginx            163189 163163 163189 15.58   0
nginx            163169 163163 163169 15.58   0
nginx            163178 163163 163178 15.58   0
nginx            163180 163163 163180 15.58   0
nginx            163172 163163 163172 15.58   0
nginx            163165 163163 163165 15.59   0
nginx            163191 163163 163191 15.58   0
nginx            163168 163163 163168 15.59   0
nginx            163186 163163 163186 15.59   0
nginx            163174 163163 163174 15.59   0
nginx            163177 163163 163177 15.59   0
nginx            163182 163163 163182 15.59   0
nginx            163166 163163 163166 15.59   0
nginx            163164 163163 163164 15.59   0
nginx            163183 163163 163183 15.59   0
nginx            163167 163163 163167 15.59   0
nginx            163170 163163 163170 15.59   0
nginx            163181 163163 163181 15.59   0
nginx            163187 163163 163187 15.59   0
nginx            163175 163163 163175 15.59   0
nginx            163185 163163 163185 15.59   0
nginx            163163 162602 163163 15.60   0
gunicorn         163154 163151 163154 16.65   signal 15 (TERM)
gunicorn         163155 163151 163155 16.63   signal 15 (TERM)
gunicorn         163151 162602 163151 16.90   0
valkey-server    162649 162602 162649 21.35   0
python3          163862 163735 163862 0.08    0
```